src/common/summarize-profile.js

```diff
@@ -70,10 +81,10 @@ const categories = [
   [match.prefix, 'Interpret(', 'script.execute.interpreter'],
 ];
 
-export function summarizeProfile(profile) {
+export function summarizeProfile(profile: Profile) {
   return timeCode('summarizeProfile', () => {
-    const categories = categorizeThreadSamples(profile);
-    const rollingSummaries = calculateRollingSummaries(profile, categories);
+    const categories: Array<string[]> = categorizeThreadSamples(profile);
+    const rollingSummaries:RollingSummary[] = calculateRollingSummaries(profile, categories);
```

Comment:
Added a rule for it. Additional rules listed out here: https://github.com/gajus/eslint-plugin-flowtype#eslint-plugin-flowtype-rules-type-id-match